core/src/main/java/org/frankframework/http/rest/ApiListener.java

```diff
@@ -124,6 +126,13 @@ public enum HttpMethod {
 	private @Getter JwtValidator<SecurityContext> jwtValidator;
 	private @Setter ServletManager servletManager;
 
+	private @Getter HttpEntityType responseType;
+	private @Getter String responseMultipartXmlSessionKey;
+	private @Getter String responseResultBodyPartName;
```

Comment:
Is 'firstPartName' oid niet duidelijker?

Reply:
Maar waar haalt hij dan dat eerste part vandaan?
Het gaat om het pipeline-result, wat de naam wordt van het bodypart waar dat in gaat, toch?

`firstPartName` is wel hoe het wordt genoemd in de HttpSender maar ik weet niet of het duidelijker is.

Reply:
Ik denk wel dat het handiger is de benaming het zelfde te houden.